### DISCUSSION TITLE
set hour12 or hourCycle in input type time

### DISCUSSION THREAD

**Discussion by lucapu88** [bootstrap-vue/bootstrap-vue#6456] [Q&A]
https://bootstrap-vue.org/docs/components/form-timepicker#form-timepicker
https://bootstrap-vue.org/docs/components/time#internationalization

How is it possible to insert the hourCycle h24 or hour12 false?
I tried with 
```
          <b-form-input
                    id="example-input"
                    v-model="timeStart"
                    type="time"
                    hour12="false"
                    hourCycle="h24"
           ></b-form-input>
```
but he doesn't work.

**Comment by jacobmllr95** [comment]
When you want to define a non-string prop make sure to add a `:` before the prop name.

`hourCycle` is no prop. it's state is just contained by the `context` object:
https://bootstrap-vue.org/docs/components/time#understanding-the-hourcycle

See this CodeSanbox for a working example:
https://codesandbox.io/s/modern-platform-fe91y?file=/App.vue

**Reply by lucapu88** [reply]
It can also be solved by adding the `locale="it"`?
